### ISSUE TITLE
DatePicker modal jumps onLoad

### ISSUE THREAD

**Issue by MartijnWeijman** [web-ridge/react-native-paper-dates#309]
I have an issue that when I open the DatePickerModal it goes to the Top of the screen and then jumps into the correct position. I'm using for the emulator a Google Pixel 6 Pro and on my physicial device (Flip 3) it has the same issue. Hopefully anyone could help me with this!

<img src="https://github.com/web-ridge/react-native-paper-dates/assets/90606995/a45bcf28-1bdf-429a-addb-3a0403609b20" width="250" height="600"/>

**Comment by MartijnWeijman** [comment]
Here is a example repo of how I use it in my App: https://github.com/MartijnWeijman/DatePickerExample

**Comment by SoyDiego** [comment]
Same issue here!
I came here to report the issue but I'm glad you reported.
Hope they can fix soon:
![error](https://github.com/web-ridge/react-native-paper-dates/assets/19764334/6d4c2f3d-fead-4bec-9c44-67e9ae557a3b)

**Comment by iM-GeeKy** [comment]
PRs are welcome!

**Comment by SoyDiego** [comment]
> PRs are welcome!

I would like to help but I don't find the file `CONTRIBUTING.md` or a guide to explain how to do the first steps.

**Comment by iM-GeeKy** [comment]
> > PRs are welcome!
> 
> I would like to help but I don't find the file `CONTRIBUTING.md` or a guide to explain how to do the first steps.

https://web-ridge.github.io/react-native-paper-dates/docs/contributing

**Comment by SoyDiego** [comment]
Are you sure the documentation is ok?
I forked the project and I tried to do the first steps and nothing.

![problem](https://github.com/web-ridge/react-native-paper-dates/assets/19764334/fd2f9cf9-78f9-4c3a-925f-52981dabb65f)

**Comment by SoyDiego** [comment]
Yes, I installed yarn.
I can see the files in my VSCode.
I tried using `npm install` and I have `node_modules` but I cannot start the project.

![image](https://github.com/web-ridge/react-native-paper-dates/assets/19764334/dfef5c1f-4743-450f-bc36-215305f6ad62)

Very weird

**Comment by SoyDiego** [comment]
I was in my work before with Windows.
Now in my home with Linux I followed the docs and I could start the project.
Thanks and sorry for write in this post all this things.

I hope we can fix the problem

**Comment by iM-GeeKy** [comment]
> I was in my work before with Windows. Now in my home with Linux I followed the docs and I could start the project. Thanks and sorry for write in this post all this things.
> 
> I hope we can fix the problem of the modal

No problem. I don't have a windows machine so I probably wasn't much help. I'll be around if you have any questions though.

**Comment by SoyDiego** [comment]
I have checked almost all the componentes touching the styles and I couldn't find the problem.
I realized that it's not the modal, it's some component behind it that causes the content behind it to jump.
I reduced the height of the modal to show:

![dates problem](https://github.com/web-ridge/react-native-paper-dates/assets/19764334/381cced1-c27c-41a6-80e8-7ed32d060fa8).

Maybe with this information can help me or guide me in the correct direction to try to fix it.

Thanks again
